tests/logging_tests/tests.py

```diff
@@ -513,3 +516,60 @@ def test_extra_args(self):
                 }},
             )]
         )
+
+
+class LogFormattersTests(SetupDefaultLoggingMixin, SimpleTestCase):
+    def test_server_formatter_styles(self):
```

Comment:
Yes, except `LoggingFiltersTest` and `CallbackFilterTest` in same module. What do you think about fixing formatting in this test cases, or it should be done in separate commit?